src/main/scala/rx/lang/scala/Observable.scala

```diff
@@ -5160,12 +5235,63 @@ object Observable {
     val jCombineFunction = new rx.functions.FuncN[R] {
       override def call(args: java.lang.Object*): R = combineFunction(args.map(_.asInstanceOf[T]))
     }
     toScalaObservable[R](rx.Observable.combineLatest[T, R](jSources, jCombineFunction))
   }
+
+  /**
+   * Combines an [[scala.collection.Iterable Iterable]] of source [[Observable]]s by emitting an item that aggregates the latest
+   * values of each of the source [[Observable]]s each time an item is received from any of the source [[Observable]]s, where this
+   * aggregation is defined by a specified function.
+   *
+   * $supportBackpressure
+   *
+   * $noDefaultScheduler
+   *
+   * @tparam T the common base type of source values
+   * @tparam R the result type
+   * @param sources the [[scala.collection.Iterable Iterable]] of source [[Observable]]s
+   * @param combineFunction the aggregation function used to combine the items emitted by the source [[Observable]]s
+   * @return an [[Observable]] that emits items that are the result of combining the items emitted by the source
+   *         [[Observable]]s by means of the given aggregation function
+   * @see <a href="http://reactivex.io/documentation/operators/combinelatest.html">ReactiveX operators documentation: CombineLatest</a>
+   */
+  def combineLatest[T, R](sources: Iterable[Observable[T]])(combineFunction: Seq[T] => R): Observable[R] = {
+    val jSources = sources.map(_.asJavaObservable).asJava
+    val jCombineFunction = new rx.functions.FuncN[R] {
+      override def call(args: java.lang.Object*): R = combineFunction(args.map(_.asInstanceOf[T]))
+    }
+    toScalaObservable[R](rx.Observable.combineLatest[T, R](jSources, jCombineFunction))
+  }
+
+  /**
+   * Combines an [[scala.collection.Iterable Iterable]] of source [[Observable]]s by emitting an item that aggregates the latest
+   * values of each of the source [[Observable]]s each time an item is received from any of the source [[Observable]]s, where this
+   * aggregation is defined by a specified function and delays any error from the sources until all source [[Observable]]s terminate.
+   *
+   * $supportBackpressure
+   *
+   * $noDefaultScheduler
+   *
+   * @tparam T the common base type of source values
+   * @tparam R the result type
+   * @param sources the [[scala.collection.Iterable Iterable]] of source [[Observable]]s
+   * @param combineFunction the aggregation function used to combine the items emitted by the source [Observable]]s
+   * @return an [[Observable]] that emits items that are the result of combining the items emitted by the source
+   *         [[Observable]]s by means of the given aggregation function
+   * @see <a href="http://reactivex.io/documentation/operators/combinelatest.html">ReactiveX operators documentation: CombineLatest</a>
+   */
+  def combineLatestDelayError[T, R](sources: Iterable[Observable[T]])(combineFunction: Seq[T] => R): Observable[R] = {
```

Comment:
Yeah, I guess `Observable.combineLatestDelayError` is better than  `ErrorDelayingObservable.combineLatest` (unless there will be many more static `xxxDelayError` methods, maybe)